cmd/root.go

```diff
@@ -73,7 +74,15 @@ func initConfig() {
 
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
-		fmt.Fprintln(os.Stderr, "Using config file:", viper.ConfigFileUsed())
+	err := viper.ReadInConfig()
+	if err == nil {
```

Comment:
A few small nits that I think clean this up a bit more:
 - I wouldn't worry about checking the error from printing the output. Despite being good practice to check errors, the err from `fmt.Print...` type messages seem to be an exception.
 - Rather than checking for `err==nil` then checking `err!=nil && ...` just do a switch statement on the error like this so that all the error checking is in one block:
```
switch {
  case !errors.As(...):
  case err != nil:
  default:
}
```